### ISSUE TITLE
Responsive?

### ISSUE THREAD

**Issue by johnstonian** [10up/flexibility#60]
Is there a known issue with rendering a flexbox layout when the browser is resized in IE9?

Pardon me if this has been asked before. And maybe I'm doing it wrong. I tried the test files, namely tests/test-widths.html in IE9. The flexbox doesn't adjust when the browser window is resized. Has anyone else experienced this issue?

**Comment by Steveeeie** [comment]
Yes, I'm experiencing the same issue, anyone know a work around?
